web_src/js/components/RepoActionView.vue

```diff
@@ -345,8 +348,17 @@ export function initRepositoryActionView() {
       button.job-brief-rerun:hover{
         transform:scale(130%);
       };
+      a.job-brief-link {
+        flex-grow: 1;
+        display: flex;
+        span {
+          margin-right: 8px;
+          display: flex;
+          align-items: center;
+        }
+      }
     }
-    a.job-brief-item:hover {
+    div.job-brief-item:hover {
```

Comment:
```suggestion
    .job-brief-item:hover {
```

Generally speaking, CSS selectors should not include the element name if the class is already unique.